examples/resources/env0_configuration_variable/resource.tf

```diff
@@ -2,3 +2,13 @@ resource "env0_configuration_variable" "example" {
   name  = "ENVIRONMENT_VARIABLE_NAME"
   value = "example value"
 }
+
+resource "env0_configuration_variable" "drop_down" {
+  name  = "ENVIRONMENT_VARIABLE_NAME"
```

Comment:
this won't really work because this file already has `ENVIRONMENT_VARIABLE_NAME`, I know that's a small semantic but let's try to keep the examples working.

Reply:
ac80073f